src/test/java/core/basesyntax/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationStrategyTest {
+    private static OperationHandler operationHandler;
+    private static OperationStrategy operationStrategy;
+    private static Map<String, OperationHandler> handlerMap;
+
+    @BeforeClass
+    public static void beforeClass() {
+        handlerMap = new HashMap<>();
+    }
+
+    @Before
+    public void setUp() {
+        operationStrategy = new OperationStrategy(handlerMap);
```

Comment:
it can be done in BeforeClass as well

Reply:
I fix it, but I don't understand why I can't see this changes here...